Formula/jigdo.rb

```diff
@@ -20,7 +21,7 @@ class Jigdo < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "berkeley-db"
+  depends_on "berkeley-db@4"
```

Comment:
GNU AGPL is not compatible with GPL-2.0-only (See https://www.gnu.org/licenses/license-list.en.html#AGPLv3.0). Can also see that GPL-3.0 library cannot be used with GPL-2.0-only work (https://www.gnu.org/licenses/gpl-faq.en.html#AllCompatibility).

Alternatively can disable feature which is what MacPorts does (https://github.com/macports/macports-ports/blob/master/net/jigdo/Portfile#L31). It does impact cache feature so need to analyze pros/cons:
```
  --without-libdb         Don't use libdb3 (it's necessary for jigdo-file's cache)
```